lib/ansible/modules/cloud/ovirt/ovirt_vms.py

```diff
@@ -444,8 +447,36 @@ def __get_template_with_version(self):
 
         return template
 
+    def __get_storage_domain(self):
+        storage_domain = None
+        if self._module.params['storage_domain']:
+            storage_domains_service = (
+                self._connection.system_service().storage_domains_service()
+            )
+            storage_domain = (
+                storage_domains_service.list(
+                    search='name=%s' % self._module.params['storage_domain']
+                )
+            )
+            if storage_domain:
+                storage_domain = storage_domain[0]
+
+        return storage_domain
+
+    def __get_disk_attachments(self, template):
+        template_service = self._connection.system_service().templates_service().template_service(template.id)
+
+        disk_attachments = self._connection.follow_link(
+            template_service.get().disk_attachments
+        )
+        disk = disk_attachments[0].disk
```

Comment:
Hmm, what if the template will have more disks. We have to somehow distinguish which one should go where...